experiments/multilingual_fineweb2_hq/download_and_tokenize_fineweb2_hq.py

```diff
@@ -64,11 +69,10 @@ def tokenize_fineweb2hq_steps(*, base_path="tokenized/", tokenizer=llama3_tokeni
                 tokenizer=versioned(tokenizer),
                 cache_options=CacheOptions(num_shard_groups=256),
             ),
-            pip_dependency_groups=["sentencepiece"],
         )
-        steps.append(step)
+        steps[os.path.join("fineweb2_hq", split)] = step
```

Comment:
nit: f"fineweb_hq/{split}" -- i don't think we want to end up with a "\" if some tries this on windows